io.openems.edge.evse.chargepoint.alpitronic/src/io/openems/edge/evcs/alpitronic/Config.java

```diff
@@ -33,6 +34,9 @@
 
 	@AttributeDefinition(name = "Maximum hardware power", description = "Maximum charging power of the Charger in W.", required = true)
 	int maxHwPower() default 75000;
+	
+	@AttributeDefinition(name = "Phase rotation", description = "The way in which the phases are physically rotated.")
+	PhaseRotation phaseRotation() default PhaseRotation.L1_L2_L3;
```

Comment:
Do we need `PhaseRotation" for a DC charger? They are anyways always three-phase-symmetric, right?